src/main/java/com/customerApi/api/controllers/CustomerDeleteController.java

```diff
@@ -0,0 +1,67 @@
+package com.customerApi.api.controllers;
+
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+
+import com.customerApi.api.modals.Customer;
+import com.customerApi.api.services.CustomerDeleteService;
+
+
+
+
+@Controller
+public class CustomerDeleteController {
+	
+	private static final Logger logger = LoggerFactory.getLogger(CustomerDeleteController.class);
+	
+	@Autowired
+	CustomerDeleteService service;
+
+	@DeleteMapping("/deleteCustomer/{id}")
+	public ResponseEntity<Customer> deleteCustomerById(@PathVariable ("id") int id)
+	{
+		logger.info("Redirect to delete service "+id);
+				
+		boolean result = service.deleteCustomerById(id);
+		 logger.info("Record delete status : "+result);
```

Comment:
The proper way of printing log is below, its an example
_For example, if you want to print 1 value"_
logger.info("Customer Name {}, sign-in in", customer.getName());

_For example, if you want to print 2 values or multiple"_
logger.info("Customer First Name {}, last name {}, sign-in in", customer.getFirstName(), customer.getLastName());